README.md

```diff
@@ -0,0 +1,21 @@
+# Flathub
```

Comment:
You probably want to rebase on top of new-pr branch to remove the unnecessary files & commits history.

Reply:
Yeah, that looked strange. Rebased it.